src/Utilities/cachingoptimizer.jl

```diff
@@ -287,19 +324,52 @@ function MOI.add_variables(m::CachingOptimizer, n)
     return vindices
 end
 
+
+"""
+    _add_bridges_if_needed(f::Function, m::CachingOptimizer)
+
+Adds bridges if it will allow `m.optimizer` to support `f(m)`.
+
+Returns `true` if bridges are added and `false` otherwise.
+"""
+function _auto_bridge_support(f::Function, model::CachingOptimizer)
+    if !f(model.model_cache)
```

Comment:
It's unclear what `f` is supposed to be. Here, it looks like it's supposed to return a bool indicating something, but the docstring seems to suggest that it returns something to be bridged.